spec/system/consumer/account_spec.rb

```diff
@@ -111,5 +111,16 @@
         expect(page).to have_content I18n.t(:you_have_no_orders_yet)
       end
     end
+
+    context "as a disabled user" do
+      before do
+        user.disabled = '1'
```

Comment:
Wow, I just learned that we don't have to persist the user. `Warden::Test::Helpers#login_as` sets the user object in memory and unless we are doing other queries that need the database, we can use an in-memory user for authentication in specs!

That line here doesn't save to the database but we don't need to in this spec.